scenario-specs/22-dao-narrative-building.md

```diff
@@ -0,0 +1,46 @@
+
+People should not be required to read through a long and potentially noisy chat log to see what's going on in a DAO. If distributed governance protocols can scale (ex: DAOstack's Holographic Consensus), the the tools for communicating and consuming communications need to scale as well.  
+
+## Example Scenarios  
+### Chat To Archive  
+**Goals: Declutter Chat, Grow Knowledge Archive**  
+**Description:** 5 group members are discussing the specifics of a new feature in chat, and they want to retain what was discussed. They select the ~50 messages and export them to an archive. Here they can further collaborate on documenting this vision, maybe through a collaborative document editor or a forum. The original messages could be retained as the "source" material within the archive.  
+
+Additionally you could:  
+* Incentivize contributions to the archive, scaling reward based on community response.  
+* Have the chat history clear after N days, further enforcing this workflow of retaining valuable communication.  
+* Replace the original messages in chat w/ a link to the new archive item, further decluttering chat.  
+
+### Call To Archive  
+**Goals: Retain Call Activity, Grow Knowledge Archive**  
+**Description:** Every week the DAO has a call where they discuss the past week. Recordings are made and uploaded to the archive. Attendees are recorded. Notes are taken and appended to the recording. Tags within the video timeline are added for easy reference.  
+
+### Group & Summarize  
+**Goals: Declutter Chat**  
+**Description:** 5 group members are discussing something, trying to form consensus. Once consensus is reached, they select the ~50 messages, and summarize them with "we decided SOLUTION was best because REASON". The original messages would still be retained if someone wanted to click through and see how the summarization was reached.  
+
+Governance over the summarization would be ideal to ensure people's thoughts are not misrepresented. Some sort of incentivized meta game could be developed to hasten and secure this process.  
+
+### Notification Replay  
+**Goals: Catch Up Quickly, Time Contextualized Activity**
+**Description:** Jill wakes up Monday morning, and wants to catch up on what happened the past 2 days over the weekend. She opens up "Notification Replay", points it at: Discord, Telegram, GitHub, the DAO, and the DAO's archive. She then scrubs through and is able to see notifications of activity that occured on all the different platforms. She can then select one and dig in deeper.  
+
+## Supporting Data  
+TODO: Show supporting data to help explain: why this scenario is important, what this scenario is trying to solve, who this scenario is mainly for.  
+
```

Comment:
We could run a test about a group of people who are trying to throw events. Maybe @oshimony and I can work through this in an offchain scenario